server/src/main/java/com/thoughtworks/go/config/update/UpdatePipelineConfigCommand.java

```diff
@@ -74,11 +82,11 @@ public boolean canContinue(CruiseConfig cruiseConfig) {
     }
 
     private boolean canEditPipeline() {
-        return goConfigService.canEditPipeline(pipelineConfig.name().toString(), currentUser, result, getPipelineGroup());
+        return goConfigService.canEditPipeline(pipelineConfig.name().toString(), currentUser, result, getExistingPipelineGroupName());
```

Comment:
We need to check if user is admin of the new group to add the pipeline right?

Reply:
Yes, we need to do. Will add that in the `update` method of the Command.